tests/sentry/integrations/github/tasks/test_open_pr_comment.py

```diff
@@ -1178,9 +1181,9 @@ def test_comment_workflow_missing_integration(
         mock_reverse_codemappings,
         mock_pr_filenames,
     ):
-        # invalid integration id
-        self.gh_repo.integration_id = 0
-        self.gh_repo.save()
+        # inactive integration
+        with assume_test_silo_mode(SiloMode.CONTROL):
```

Comment:
This is sort of an outdated way to do this. You can use `assume_test_silo_mode_of(Integration)` to infer which silo a specific model (or multiple models) should be using instead which is a bit cleaner.

Reply:
fixed!